### PR TITLE
Make jssrc2cpg default

### PR DESCRIPTION
**Pull request by pr0me**
Regarding issue #1468.

This commit makes the necessary changes to register `jssrc2cpg` as the default frontend for javascript projects.

### CODE REVIEW

**Comment by fabsx00** [comment]
:+1: let's merge this when we feel comfortable with jssrc2cpg.

File: console/src/main/scala/io/joern/console/cpgcreation/package.scala [hunk 1]
```diff
@@ -74,7 +74,7 @@ package object cpgcreation {
       case f if f.endsWith(".csproj") || f.endsWith(".cs") => Some(Languages.CSHARP)
       case f if f.endsWith(".go") || Set("gopkg.lock", "gopkg.toml", "go.mod", "go.sum").contains(f) =>
         Some(Languages.GOLANG)
-      case f if f.endsWith(".js") || f == "package.json" => Some(Languages.JAVASCRIPT)
+      case f if f.endsWith(".js") || f == "package.json" => Some(Languages.JSSRC)
```
Comment (max-leuthaeuser) [inline console/src/main/scala/io/joern/console/cpgcreation/package.scala:77]: Also `.ts`, `.jsx`, and `.tsx`.

**Comment by max-leuthaeuser** [comment]
We may want to do this this week.

**Comment by max-leuthaeuser** [comment]
Superseded by: https://github.com/joernio/joern/pull/1563

**Pull request closed without merging**